service/scheduler/test-steps/src/main/java/org/eclipse/kapua/service/scheduler/steps/JobScheduleServiceSteps.java

```diff
@@ -162,7 +162,7 @@ public void iTryToCreateSchedulerWithName(String schedulerName) throws Exception
         }
     }
 
-    @And("^I find trigger properties with name \"([^\"]*)\"$")
+    @And("^I find scheduler properties with name \"([^\"]*)\"$")
```

Comment:
Just to ask, why this has been changed from `trigger` to `scheduler`?

Reply:
Hi @Coduz,
it has been changed because of scenario logic. Are you agree with that change, or I have to back `trigger`? 

Regards,
_Ana_